app/presenters/appointment_presenter.rb

```diff
@@ -51,10 +51,20 @@ def day
   end
 
   def begin_time
-    I18n.l(begin_date, format: '%l:%M%P').strip
+    friendly_time(begin_date)
   end
 
   def end_time
-    I18n.l(end_date, format: '%l:%M%P').strip
+    friendly_time(end_date)
+  end
+
+  def friendly_time(time)
```

Comment:
Could this be extracted to an initializer? e.g. https://robots.thoughtbot.com/custom-formats-for-datetime

Reply:
I thought about this, but I've read the GOVUK guidelines that suggest using "midday" in place of "12pm", which seems like a good idea - and this wouldn't work with Rails' standard datetime formatting.